src/main/java/name/webdizz/sonar/grammar/spellcheck/GrammarDictionaryLoader.java

```diff
@@ -49,9 +49,9 @@ public SpellDictionary load() {
                         InputStream inputStream = this.getClass().getResourceAsStream(dictionaryPath);
 
                         BufferedReader dictionaryReader = null;
-                        dictionaryReader = Files.newReader( new File( dictionaryPath ), Charsets.UTF_8 );
+                        dictionaryReader = new BufferedReader(new InputStreamReader(inputStream, "UTF-8"));

```

Comment:
Could we still use Guavas's high level API instead of dealing with IO at low level?